packages/publisher/gcs/src/PublisherGCS.ts

```diff
@@ -0,0 +1,90 @@
+import path from 'path';
+
+import { PublisherBase, PublisherOptions } from '@electron-forge/publisher-base';
+import { Storage } from '@google-cloud/storage';
+import debug from 'debug';
+
+import { PublisherGCSConfig } from './Config';
+
+const d = debug('electron-forge:publish:gcs');
+
+type GCSArtifact = {
+  path: string;
+  keyPrefix: string;
+  platform: string;
+  arch: string;
+};
```

Comment:
question: Can we combine this with `S3Artifact` into a common `StaticArtifact` type? Not exactly sure where it would go yet. Feedback welcome.